app/pages/organization/stats/components/ProjectStats/ProjectStatsContainer.jsx

```diff
@@ -0,0 +1,123 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+import getWorkflowsInOrder from '../../../../../lib/get-workflows-in-order';
+import ProjectStats from './ProjectStats';
+
+class ProjectStatsContainer extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      projectStats: new Map()
```

Comment:
`new Map()` is potentially an IE11 issue, although it appears to work through BrowserStack